src/current/v23.1/why-cockroachdb.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Why CockroachDB?
+summary: What is CockroachDB? How does it work? What makes it different from other databases?
+tags: postgres, cassandra, google cloud spanner
+toc: false
+docs_area: get_started
+---
+
+{% include {{ page.version.version }}/faq/what-is-crdb.md %}
```

Comment:
I think we need a real standalone page here. I realize this increases scope, but I think with a relatively small amount of additional work we can produce page that has higher impact. Upon review, using the FAQ section just doesn't cut it here.

I shared a Google doc in Slack with some content ideas.

Reply:
Updated.